x/sequencer/keeper/msg_server_create_sequencer_test.go

```diff
@@ -55,45 +49,63 @@ func (suite *SequencerTestSuite) TestMinBond() {
 			bond:          sdk.NewCoin(bond.Denom, bond.Amount.Quo(sdk.NewInt(2))),
 			expectedError: types.ErrInsufficientBond,
 		},
+		{
+			name:          "wrong bond denom",
```

Comment:
also this test seems to test the wrong thing - i.e fail on invalid coin denom vs valid but not what we expected (i.e wrong).

Reply:
changed the test a bit